packaging/Dockerfile

```diff
@@ -5,7 +5,9 @@ LABEL maintainer="k6 Developers <developers@k6.io>"
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update -y && \
-    apt-get install -y apt-utils createrepo curl git gnupg2 python3 unzip
+    apt-get install -y apt-utils createrepo curl git gnupg2 python3-pip unzip
+
+RUN pip3 install s3cmd
```

Comment:
I'll change this to pin and expose `S3CMD_VERSION` just like `AWSCLI_VERSION` is exposed now.

Reply:
Done in a35bbfab.